do: make REALLY clear how to use setAlarm (#15685)
cloudflare/cloudflare-docs: content/durable-objects/api/alarms.md

Notably: 

* Each Durable Object instance is able to schedule a single alarm at a time by calling `setAlarm()`.

> * Once an alarm runs, you will need to call `setAlarm()` to schedule an alarm.

Reviewer: ```suggestion
* Once an alarm runs, you will need to call `setAlarm()` to schedule another alarm.
```
an -> another?

> * Alarms do not automatically run on a recurring basis as they are set to run at a specific instance in time (milliseconds since the UNIX epoch).

Reviewer 2: ```suggestion
* Alarms can only be set to run at a specific instance in time (milliseconds since the UNIX epoch); they cannot be configured to run automatically on a recurring basis.
```

* Alarms have guaranteed at-least-once execution and are retried automatically when the `alarm()` handler throws.

> * Retries are performed using exponential backoff starting at a two second delay from the first failure with up to six retries allowed.

Reviewer 2: Not sure if it also might be useful to mention here:  IIRC, Matt had some "best practices" advice that if an `alarm()` handler needs to call `setAlarm()` to schedule itself to run again later, it's best to make the `setAlarm()` call as late in the handler as possible.

I think the logic is that if a `setAlarm()` call succeeds early in a handler that later throws an exception, the new alarm time effectively suppresses the sequence of automatic retries?

  }
  async alarm() {
    // The alarm handler will be invoked whenever an alarm fires.
    // You can use this to do work, read from the Transactional Storage API or make HTTP calls

    // Use this.storage.setAlarm() from within this handler to set another alarm to run in the

>     // future: alarms do not run on a schedule, but (only) at the specific instance in time

Reviewer 3: remove?
